src/protocols/secure_channel/PASESession.h

```diff
@@ -213,11 +210,7 @@ class DLL_EXPORT PASESession : public Messaging::UnsolicitedMessageHandler,
     // mNextExpectedMsg is set when we are expecting a message.
     Optional<Protocols::SecureChannel::MsgType> mNextExpectedMsg;
 
-#ifdef ENABLE_HSM_SPAKE
-    Spake2pHSM_P256_SHA256_HKDF_HMAC mSpake2p;
-#else
     Spake2p_P256_SHA256_HKDF_HMAC mSpake2p;
```

Comment:
So in the new world, how does this end up using the HSM code?

Reply:
Examples / applications need to build with option "chip_crypto == "platform" and provide the required crypto implementation. ( In this case it will be from "\src\platform\nxp\crypto\se05x\" )